README.md

```diff
@@ -7,10 +7,12 @@ Deploy CARTO in a self hosted environment. It is provided in two flavours:
 
 ## Databases
 
-Both flavours are recomended to be installed using external and managed databases (Postgres and Redis). The versions recommended are:
+Both flavours will need two databases in order to store the metadata of the users.
```

Comment:
I removed that part of the `metadata of the users` in the past. I think it creates more noise than anything else.

```suggestion
Both self hosted flavours need two databases (Postgres and Redis) in order to work.
```